lib/cfg-lexer.c

```diff
@@ -818,10 +818,9 @@ cfg_lexer_lex(CfgLexer *self, YYSTYPE *yylval, YYLTYPE *yylloc)
           *yylloc = self->include_stack[self->include_depth].lloc;
           tok = token->type;
           if (token->type == LL_TOKEN)
-            {
-              tok = token->token;
-              injected = TRUE;
-            }
+            tok = token->token;
```

Comment:
Sure, thanks.
I'm going to push some refactor steps after @kira-syslogng's tests are completed.